packages/botonic-core/src/router.js

```diff
@@ -130,9 +131,8 @@ export class Router {
       if (pathParams.length > 1) {
         return pathParams[1]
       }
-    } catch (e) {
-      return undefined
-    }
+    } catch (e) {}
```

Comment:
Shouldn't this `catch` say something? At least on a log?

Reply:
Yes, I agree

Reply:
@marcpdw @MarcosCA thanks, I did a modification so that it should never fail, added log and unit tests :-) merging to avoid so many branches, but as usual open to feedback